File: Python/Programmars/Level_2/최댓값과 최솟값.py
def solution(s):
    s = list(map(int, s.split()))
    return f'{min(s)} {max(s)}'


print(solution("1 2 3 4"))  # "1 4"
print(solution("-1 -2 -3 -4"))  # "-4 -1"
print(solution("-1 -1"))  # "-1 -1"

File: JavaScript/Programmars/Level_2/최댓값과 최솟값.js
const solution = (s) => {
  s = s.split(' ').map((n) => +n);
  return `${Math.min(...s)} ${Math.max(...s)}`;
};

// 250417
function solution(s) {
  const numbers = s.split(' ').map((item) => Number(item));

  return `${Math.min(...numbers)} ${Math.max(...numbers)}`;
}

console.log(solution('1 2 3 4')); // "1 4"
console.log(solution('-1 -2 -3 -4')); // "-4 -1"
console.log(solution('-1 -1')); // "-1 -1"
